LICENSE.md

```diff
@@ -32,6 +32,10 @@ You don't have to [contribute](#contribute) any change, addition, or other softw
 
 3.  You don't develop, operate, or analyze software with it for anyone outside the team developing it.
 
+## Reverse Engineering
+
+You may use this software to operate and analyze software that you can't [contribute](#contribute) to develop alternatives that you can and do [contribute](#contribute).
```

Comment:
I hit a snag partway through reading this: I try to read “contribute to” as a phrasal verb, then realize right after that it’s “contribute [in order] to”.

Reply:
I had "in order to" before. I'll add it back. Thanks!

Reply:
Addressed in 37fb535589417681ba4b866b5032796769432c0a